CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Changed
+- Clarified a comment in `DefaultSimulcastUplinkPolicy`.
```

Comment:
I do not think you need a CHANGELOG update for changes that do not effect builders.

Reply:
@ltrung can you verify?

Reply:
Unfortunately as long as you touch src folder you need CHANGELOG :D